src/parser/jobs/mnk/modules/index.ts

```diff
@@ -17,12 +16,11 @@ export default [
 	Demolish,
 	Fists,
 	Forms,
-	GreasedLightning,
 	MnkAoE,
 	OGCDDowntime,
 	PerfectBalance,
+	RiddleOfEarth,
 	RiddleOfFire,
-	Speedmod,
```

Comment:
I can see the delete of `GreasedLightning`, but not `Speedmod` - is it sitting around unused atm?

Reply:
I kept the file in for now, since I intend to re-add Speedmod eventually. The way it is atm (operating off statuses) didn't really fly when the statuses don't exist anymore, and it was inaccurate anyway. I tried to get it to use a scalar set by the specific gauge state but it was kinda derpy so I just disabled it for now to avoid confusing users.